<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inventario</class>
 <widget class="QDialog" name="Inventario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/lista</normaloff>:/lista</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="5">
    <widget class="QTableWidget" name="outProductos"/>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnAgregar">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(43, 255, 32);</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnEliminar">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 78, 66);</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnGuardar">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Guardar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
